docs/NLUGitHubActions.md

```diff
@@ -0,0 +1,78 @@
+## GitHub Actions workflow using NLU.DevOps CLI tool
+
+This document covers setting up a CI pipeline for LUIS using GitHub Actions and NLU.DevOps.
+The workflow will be the following: 
+1. create, train, publish the LUIS model using sample utterances
+2. send a test set to created LUIS model
+3. evaluate model by comparing results received from LUIS with expected values
+4. delete the model from the portal
+
+Supply a name for the action and set it up to trigger on pull requests.
+
+```
+name: CINLU
+
+on: [pull_request]
+```
+
+1. Install NLU.DevOps CLI tool on GitHub agent.
```

Comment:
Rather than using ordered list items, maybe we should just break these into sub-sections. I.e., this one could be called:

```
### Installing NLU.DevOps on the GitHub Actions agent
```